Automattic/wp-calypso
# Add/mechanism to check for session tracking availability

## Proposed Changes

* Add mechanism to check for STS (Session Tracking Sensitive) regions
* Use new mechanism to block hotjar from loading

## Testing Instructions

1. Checkout this branch locally or use the calypso green live link
2. Go to `/pricing?flags=ad-tracking`
3. If you are in one of the STS states (listed in is-region-in-sts-zone.ts), update your `region` cookie to a non-STS state (ex. Idaho)
4. Refresh the page and search the DOM for 'hotjar' and make sure the script loaded
![image](https://github.com/Automattic/wp-calypso/assets/65001528/39022d12-4340-4a30-9165-8e62cc48524e)
5. Now change your `region` cookie to an STS state (ex. Washington) and refresh the page again. Make sure the script does not load
![image](https://github.com/Automattic/wp-calypso/assets/65001528/90b220a4-f68c-4401-8b55-17db8282f78e)

## Pre-merge Checklist
